attributes/default.rb

```diff
@@ -3,11 +3,9 @@
 default['yum']['kernel-osuosl']['enabled'] = true
 default['yum']['kernel-osuosl']['gpgcheck'] = true
 default['yum']['kernel-osuosl']['gpgkey'] = 'https://ftp.osuosl.org/pub/osl/repos/yum/RPM-GPG-KEY-osuosl'
-case node['kernel']['machine']
-when 'ppc64le'
-  default['yum']['kernel-osuosl']['baseurl'] =
-    'https://ftp.osuosl.org/pub/osl/repos/yum/openpower/centos-$releasever/ppc64le/kernel-osuosl/'
-else
-  default['yum']['kernel-osuosl']['baseurl'] =
-    'https://ftp.osuosl.org/pub/osl/repos/yum/$releasever/kernel-osuosl/$basearch/'
-end
+default['yum']['kernel-osuosl']['baseurl'] = case node['kernel']['machine']
```

Comment:
Can you bring the case statement down one line and then indent it? That should still make cookstyle happy.